data-ingestion-apis/opentelemetry-metrics.mdx

```diff
@@ -1,42 +1,86 @@
 ---
-title: "Data Ingestions API"
+title: "Customizing Telemetry Ingestion"
+sidebarTitle: "OpenTelemetry"
 ---
 
 # Overview
-This guide provides an overview and setup instructions for Middleware’s OpenTelemetry Data Ingestion API. The OpenTelemetry Ingestion API has two endpoints: [metrics](#metrics-endpoint) and [logs](#logs-endpoint).
+We recommend using Middleware's Agent and associated SDKs to ingest data, however, more advanced users have the option to use OpenTelemetry Protocol Exporters. Each language will have a different approach which can be an adaptation from our relevant Host Agent. Instrument your OpenTelemetry code with languages that are supported by both Middleware and [OpenTelemetry](https://opentelemetry.io/docs/instrumentation/).
 
-## Resource Types
+The example below is using javascript and an abstraction from our [node agent](https://github.com/middleware-labs/agent-apm-nodejs). All other agents can be found in our [middleware-labs repo](https://github.com/middleware-labs). If you need assistance with a specific language we are happy to work with you, however, we do not have every approach documented at this time.
 
-For both endpoints, the resource type attribute groups the ingested data under the specified label on Middleware dashboards and reports. The resource type field requires one of the following options.
+# Prerequisites
 
+1. App must already be configured to collect telemetry 
 
-| **Group** | **Resource Type** |
-| :---------: | :---------------: |
-| host | host.id |
-| k8s.node | k8s.node.uid |
-|k8s.pod | k8s.pod.uid |
-| k8s.deployment | k8s.deployment.uid |
-| k8s.daemonset | k8s.daemonset.uid |
-| k8s.replicaset| k8s.replicaset.uid |
-| k8s.statefulset | k8s.statefulset.uid |
-| k8s.namespace | k8s.namespace.uid |
-| service | service.name |
-| os | os.type |
-| custom | custom |
+# Installation
 
-## Metrics Endpoint
+### Step 1: Setup Resource Attributes
 
-The Metrics endpoint lets you send custom metrics to the Middleware backend. 
+Embed OpenTelemetry Resource Attributes:
```

Comment:
Let's give a "why we're talking about this for this section" e.g. "Middleware connects via Otel using the mw.account_key which can be found [in the x section of the docs](with some link). To get started, you need to set up appropriate Resource attributes, seen below:"